go.mod

```diff
@@ -41,9 +42,12 @@ require (
 )
 
 require (
+	buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go v1.31.0-20230914171853-63dfe56cc2c4.1 // indirect
```

Comment:
We can't do this directly without a `replace` - see what we did in https://github.com/bufbuild/buf/pull/2473. I'd recommend we get #2473 merged first.

Reply:
Merged in